urbs/model.py

```diff
@@ -518,7 +524,11 @@ def create_model(data, dt=1, timesteps=None, dual=False):
         m.t, m.sto_tuples - m.sto_init_bound_tuples,
         rule=res_initial_and_final_storage_state_var_rule,
         doc='storage content initial <= final, both variable')
-
+    m.def_storage_energy_to_power = pyomo.Constraint(
+        m.sto_en_to_pow_tuples,
+        rule=def_storage_energy_to_power_rule,
```

Comment:
Maybe leaving out the 'to' might be a bit better? Just a suggestion, open for discussion.

Reply:
This suggestion is applied in the new commit.